internal/provider/launchpad_config_resource.go

```diff
@@ -154,20 +150,20 @@ func getterToModelAndProduct(ctx context.Context, diags *diag.Diagnostics, gette
 		return ls, mke
 	}
 
-	cc := ls.ClusterConfig(diags)
+	cc := ls.ClusterConfig(ctx, diags)
```

Comment:
clusterconfig was using context.Background(), but we have the TF context here, so we just pass it through now.